app/tests/integration/nimbus/test_e2e_create_experiment.py

```diff
@@ -42,3 +47,84 @@ def test_create_new_experiment(selenium, base_url):
 
     # Review
     selenium.find_element_by_css_selector("#PageRequestReview")
+
+
+def test_create_new_experiment_remote_settings(selenium, base_url):
+    experiment_name = f"name here remote {random.randint(0, 1000)}"
+
+    selenium.get(base_url)
+    home = HomePage(selenium, base_url).wait_for_page_to_load()
+    current_experiments = len(home.tables[0].experiments)
+    experiment = home.create_new_button()
+    experiment.public_name = experiment_name
+    experiment.hypothesis = "smart stuff here"
+    experiment.application = "DESKTOP"
+
+    # Fill Overview Page
+    overview = experiment.save_and_continue()
+    overview.public_description = "description stuff"
+    overview.select_risk_brand_false()
+    overview.select_risk_revenue_false()
+    overview.select_risk_partner_false()
+
+    # Fill Branches page
+    branches = overview.save_and_continue()
+    branches.remove_branch()
+    branches.reference_branch_name = "name 1"
+    branches.reference_branch_description = "a nice experiment"
+    branches.feature_config = "No Feature Firefox Desktop"
+
+    # Fill Metrics page
+    metrics = branches.save_and_continue()
+
+    # Fill Audience page
+    audience = metrics.save_and_continue()
+    audience.channel = "Nightly"
+    audience.min_version = 80
+    audience.targeting = "US_ONLY"
+    audience.percentage = 50.0
+    audience.expected_clients = 50
+    audience.save_btn()
+    review = audience.save_and_continue()
+
+    # Review and approve
+    selenium.find_element_by_css_selector("#PageRequestReview")
+    review.launch_without_preview.click()
+    review.request_review.click_launch_checkboxes()
+    review.request_review.request_launch_button.click()
+    review.approve()
+    selenium.get("http://kinto:8888/v1/admin")
+    kinto_login = Login(selenium, base_url).wait_for_page_to_load()
+    kinto_login.kinto_auth.click()
+    kinto_dashbard = kinto_login.login()
+    bucket = kinto_dashbard.buckets[-1]
+    for item in bucket.bucket_category:
+        if "nimbus-desktop-experiments" in item.text:
+            item.click()
+            break
+    record = kinto_dashbard.record
+    record.approve()
+    selenium.get(base_url)
+    # refresh until the experiment shows up
+    for attempt in range(45):
+        try:
+            home = HomePage(selenium, base_url).wait_for_page_to_load()
+            new_experiments = len(home.tables[0].experiments)
+            assert new_experiments != current_experiments
+        except AssertionError:
+            time.sleep(2)
+            selenium.refresh()
+        else:
+            print("live")
+            break
+    # Check it's live
+    home = HomePage(selenium, base_url).wait_for_page_to_load()
+    live_experiments = home.tables[0]
+    assert "live experiments" in live_experiments.table_name.lower()
+    for item in live_experiments.experiments:
+        if experiment_name in item.text:
+            print(item.text)
```

Comment:
Is this for local debugging, or do we need to keep these for CI troubleshooting reasons? (this and L118 above, which prints "live" to STDOUT).